docs/conf.py

```diff
@@ -3,13 +3,19 @@
 import datetime as dt
 import json
 import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../src/uwtools"))
 
 with open("../recipe/meta.json", "r", encoding="utf-8") as f:
     _metadata = json.loads(f.read())
 
 copyright = str(dt.datetime.now().year)
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.intersphinx"]
 html_logo = os.path.join("static", "ufs.png")
 html_theme = "sphinx_rtd_theme"
+intersphinx_mapping = {"python": ("https://docs.python.org/3", None)}
+nitpick_ignore_regex = [("py:class", r"^uwtools\..*")]
```

Comment:
There are type names in our API signatures that point to `uwtools` types for which we do not yet generate HTML docs. For now, this line suppresses related warnings/errors from `make html`.